libs/wai-utilities/src/Network/Wai/Utilities/Request.hs

```diff
@@ -17,8 +18,7 @@
 -- You should have received a copy of the GNU Affero General Public License along
 -- with this program. If not, see <https://www.gnu.org/licenses/>.
 
-{-# OPTIONS -Wno-orphans #-}
--- for "instance HasRequest Request" :(
+-- orphan instance for "instance HasRequest Request" :(
```

Comment:
This should also be moved up.

Reply:
It can't, with current Ormolu. It will move all comments below the `OPTIONS_GHC` and `LANGUAGE` pragmas.
This was changed in Ormolu recently, but is not released yet: https://github.com/tweag/ormolu/blob/master/CHANGELOG.md